mitodl/open-discussions
# Added sorting to channel and frontpage apis

#### What are the relevant tickets?
Fixes #192 

#### What's this PR do?
Adds sorting to the channel and frontpage apis

#### How should this be manually tested?
- Verify the existing frontpage and channels sort exactly the same as master
- Verify you can pass alternative sorting methods to the api directly (e.g. `/api/v0/channels/:channel_name/posts/?sort=top|hot|new`)

Note: you need a decent variety of posts/comments/users to make these sorts sufficiently different
